org.eclipse.winery.repository.ui/src/app/wineryRepository.component.ts

```diff
@@ -20,13 +21,15 @@ import { backendBaseURL } from './configuration';
         ExistService
     ]
 })
+
 /*
  * This component represents the root component for the Winery Repository.
  */
 export class WineryRepositoryComponent implements OnInit {
     // region variables
     name = 'Winery Repository';
-    isBackendAvailable = false;
+    isBackendAvailable = BackendStates.Undefined;
+    backendState = BackendStates;
```

Comment:
Please use the pluaral: `backendStates`. You only use this variable as a reference and it doesn't represent the actual backend state.